pandas/tests/generic/test_label_or_level_utils.py

```diff
@@ -130,14 +133,16 @@ def test_is_level_reference_series_axis1_error(df):
 # ---------
 def test_check_label_or_level_ambiguity_df(df_ambig, axis):
 
+    axis = df_ambig._get_axis_number(axis)
     # Transpose frame if axis == 1
-    if axis in {1, "columns"}:
+    if axis == 1:
         df_ambig = df_ambig.T
+        msg = "'L1' is both a column level and an index label"
 
-    if axis in {0, "index"}:
-        msg = "'L1' is both an index level and a column label"
     else:
-        msg = "'L1' is both a column level and an index label"
+        msg = "'L1' is both an index level and a column label"
+    
+        
```

Comment:
remove these blank lines. You get CI check errors

Reply:
Done